src/main/resources/templates/rules.html

```diff
@@ -10,6 +10,19 @@
 <div th:replace="include/navigationbar :: navigationbar"></div>
 <main role="main" class="container">
     <h1 class="pageHeading">Mastermind: Die Regeln</h1>
+    <div class="Infos">
+        <p>Genaue Infos zum Spiel Mastermind findest du hier:
+            <a class="customLink" href="https://de.wikipedia.org/wiki/Mastermind_(Spiel)">Wikipedia: Mastermind</a></p>
```

Comment:
In dem "a" tag sollte man noch `target="_blank"` hinzufügen. Zusätzlich würde ich statt `Wikipedia: Mastermind` nur `Mastermind` anzeigen lassen